driver-adapters-wasm/pg-cfpages-basic/test.sh

```diff
@@ -2,7 +2,9 @@
 
 set -eu
 
-export DEPLOYMENT_URL=$(cat deployment-logs.txt | grep -Eo "(https.*)\.pages\.dev$" --color=never)
-echo $DEPLOYMENT_URL
+echo "Temporary disabled, see https://github.com/cloudflare/workers-sdk/pull/2541"
```

Comment:
```suggestion
echo "Temporary disabled, because wrangler does not support "--node-compat" flag for pages commands yet. See https://github.com/cloudflare/workers-sdk/pull/2541"
```